### ISSUE TITLE
Accessibility:List: The “loading” information is not read, while accessing list items. 

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3482]
Description: The “loading” information is not read, while accessing list items. 

Expected: Loading information should be conveyed to the user. 

Screenshot: 

![image](https://user-images.githubusercontent.com/32538291/94648619-8cb58b00-0310-11eb-9b0a-d32f26c27dea.png)
